tensorflow/compiler/mlir/tools/kernel_gen/transforms/tf_kernel_to_llvm_pass.cc

```diff
@@ -254,6 +254,11 @@ class TFKernelToLLVMPass : public TFKernelToLLVMPassBase<TFKernelToLLVMPass> {
 
     // Populate type conversions.
     MLIRContext *ctx = m.getContext();
+    mlir::LowerToLLVMOptions llvm_opts(
+        m.getContext(),
+        DataLayout(cast<DataLayoutOpInterface>(m.getOperation())));
+    llvm_opts.overrideIndexBitwidth(64);
+    LLVMTypeConverter type_converter(ctx,llvm_opts);
```

Comment:
This looks like a rebase artifact

Reply:
Cleaned up. Remanence of hard coding 64 bit indexing options :)